runtime/src/lib.rs

```diff
@@ -774,6 +779,34 @@ impl pallet_proxy::Config for Runtime {
     type AnnouncementDepositFactor = AnnouncementDepositFactor;
 }
 
+mod vk_registration_parameters {
+    use super::*;
+
+    fn vks_key_size() -> u32 {
+        Identity::max_len::<sp_core::H256>() as u32
+    }
+    fn tickets_key_size() -> u32 {
+        Blake2_128Concat::max_len::<(AccountId, sp_core::H256)>() as u32
+    }
+    fn tickets_value_size() -> u32 {
+        VkRegistrationHoldConsideration::max_encoded_len() as u32
+    }
+    parameter_types! {
+        pub VkRegistrationBaseDeposit: Balance = deposit(4, vks_key_size() + tickets_key_size() + tickets_value_size());
```

Comment:
Couple of points here:

- Why 4 items ? You are depositing only 2 keys in the maps `vk_hash` in `Vks `and `(account, vk_hash)` in `Tickets`. 
- I'm not even sure you need to make the user pay also for storing the HoldConsideration in `Tickets` (looking around in all pallets this doesn't seem the case because either they don't use the HoldConsideration or because they embed it in the value item in the map)
- I think we can avoid overabstracting here and write explicitly all the constants (they are well known and fixed anyway), maybe just writing a comment is enough

Reply:
Changed the number of items from 4 to 2 in https://github.com/zkVerify/zkVerify/pull/178/commits/7fbf1c61868af60b81eef761e00d8ea8677a068c. For the other points, decided to keep it as it is, as discussed offline.